audit.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3.7
+
+import os
+import sys
+from subprocess import CalledProcessError, check_call
+import github3 as gh3
+
+DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I think we need a better variable for this, as it's too generic and looks like a mis-typed version of Python's `dir` function. `SCRIPT_DIR` or something like that might be a bit better?